packages/wxt/src/core/utils/building/__tests__/find-entrypoints.test.ts

```diff
@@ -668,27 +668,26 @@ describe('findEntrypoints', () => {
     expect(entrypoints[0]).toEqual(expected);
   });
 
-  it('should not allow multiple entrypoints with the same name', async () => {
+  it('should ignore multiple */index.[jt]sx? if  */index.html already exists', async () => {
     globMock.mockResolvedValueOnce([
       'options/index.html',
       'options/index.jsx',
-      'popup.html',
       'popup/index.html',
       'popup/index.ts',
       'ui.html',
     ]);
+    expect(() => findEntrypoints());
```

Comment:
this doesn't actually check anything. needs t be awaited and use [.resolves](https://vitest.dev/api/expect-typeof.html#resolves) API